src/TypeScript/EslintBridgeClient/EslintBridgeHttpWrapper.cs

```diff
@@ -55,13 +57,25 @@ public async Task<string> PostAsync(Uri serverEndpoint, object request, Cancella
             var serializedRequest = request == null
                 ? string.Empty
                 : JsonConvert.SerializeObject(request, Formatting.Indented);
-            logger.LogDebug("[eslint-bridge] Endpoint: {0}, request:{1}{2}", serverEndpoint, Environment.NewLine, serializedRequest);
+            logger.LogDebug("[eslint-bridge POST] Endpoint: {0}, request:{1}{2}", serverEndpoint, Environment.NewLine, serializedRequest);
 
             var content = new StringContent(serializedRequest, Encoding.UTF8, "application/json");
             var response = await httpClient.PostAsync(serverEndpoint, content, cancellationToken);
             var responseString = await response.Content.ReadAsStringAsync();
 
-            logger.LogDebug("[eslint-bridge] Endpoint: {0}, response:{1}{2}", serverEndpoint, Environment.NewLine, responseString);
+            logger.LogDebug("[eslint-bridge POST] Endpoint: {0}, response:{1}{2}", serverEndpoint, Environment.NewLine, responseString);
+
+            return responseString;
+        }
+
+        public async Task<string> GetAsync(Uri serverEndpoint, CancellationToken cancellationToken)
+        {
+            logger.LogDebug("[eslint-bridge GET] Endpoint: {0}{1}", serverEndpoint, Environment.NewLine);
+
+            var response = await httpClient.GetAsync(serverEndpoint, cancellationToken);
+            var responseString = await response.Content.ReadAsStringAsync();
+
+            logger.LogDebug("[eslint-bridge GET] Endpoint: {0}, response:{1}{2}", serverEndpoint, Environment.NewLine, responseString);
 
             return responseString;
```

Comment:
I looked at whether I could have a common helper method that `PostAsync` and `GetAsync` could delegate the work too, but the methods signatures are different.